thunder/clang/__init__.py

```diff
@@ -280,6 +280,13 @@ def full_like(
     return full(a.shape, fill_value, device=device, dtype=dtype)
 
 
+@clangop()
+def empty(shape: Sequence[int], *, device: DeviceLike, dtype: None | dtypes.dtype = None) -> TensorLike:
```

Comment:
For `empty`, let's require the `dtype` parameter be specified, like with `uniform` below